conjur/cli.py

```diff
@@ -11,36 +11,66 @@
 import json
 import os
 import sys
+import requests
+
+from .argparsewrapper import ArgparseWrapper
 
 from .client import Client
 
 from .version import __version__
 
-
 class Cli():
     """
     Main wrapper around CLI-like usages of this module. Provides various
     helpers around parsing of parameters and running client commands.
     """
 
-    #pylint: disable=no-self-use
+    @staticmethod
+    def main_description():
+        """
+        This method builds the header for the main screen.
+        """
+        return '''Usage:
+    conjur [global options] command [subcommand] [value] [--option=arg]'''
+
+    @staticmethod
+    def main_epilog():
+        """
+        This method builds the footer for the main help screen.
+        """
+        return '''
+To get help on a specific command, see `conjur <command> -h`'''
```

Comment:
can this be changed to
```
'''
To get help on a specific command, see `conjur <command> -h`
'''
```

Reply:
With the formatter indentation is important here. I can change to
```
return '''
To get help on a specific command, see `conjur <command> -h`
'''
```